### PR TITLE
Remove thrift client stats module

### PR DESCRIPTION
**Pull request by AlekseiS**

### CODE REVIEW

**Comment by martint** [comment]
Add some detail to the commit message explaining the rationale.

**Comment by AlekseiS** [comment]
Added commit message.

File: presto-thrift-connector/pom.xml [hunk 1]
```diff
@@ -48,11 +48,6 @@
             <artifactId>bootstrap</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>io.airlift</groupId>
-            <artifactId>json</artifactId>
```
Comment (martint) [inline presto-thrift-connector/pom.xml]: This seems unrelated to JMX stats

File: presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftModule.java [hunk 2]
```diff
@@ -64,8 +64,7 @@ public void configure(Binder binder)
         configBinder(binder).bindConfig(ThriftConnectorConfig.class);
         binder.bind(ThriftSessionProperties.class).in(Scopes.SINGLETON);
         binder.bind(ThriftIndexProvider.class).in(Scopes.SINGLETON);
-        newExporter(binder).export(PrestoThriftServiceProvider.class)
-                .as(generatedNameOf(RetryingPrestoThriftServiceProvider.class, connectorId));
+        newExporter(binder).export(PrestoThriftServiceProvider.class).as(generatedNameOf(RetryingPrestoThriftServiceProvider.class, connectorId));
```
Comment (electrum) [inline presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftModule.java]: Unrelated formatting change?

Comment (electrum) [reply presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftModule.java]: BTW, the name `RetryingPrestoThriftServiceProvider` seems quite long. Should we shorten the exported name?

Comment (AlekseiS) [reply presto-thrift-connector/src/main/java/com/facebook/presto/connector/thrift/ThriftModule.java]: The change is indeed unrelated, but I hesitated to put it in a separate commit. I removed it for now.

File: presto-thrift-connector/pom.xml (hunk 1 above)
Comment (AlekseiS) [reply presto-thrift-connector/pom.xml]: JsonModule was removed, so dependency was removed as well. ClientStats was the only possible module that could require it. In any case, removing it as part of cleaning up extra modules seems fine to me.

**Comment by AlekseiS** [comment]
@martint , @electrum , any other comments before I can merge it?

File: presto-thrift-connector/pom.xml (hunk 1 above)
Comment (martint) [reply presto-thrift-connector/pom.xml]: Remove JsonModule and this dep in a separate commit (could be in same PR), since it's unrelated to removing the stats.

**Comment by AlekseiS** [comment]
Split PR in two commits: remove thrift client stats and remove json module
